### DISCUSSION TITLE
Dicuss technolgy stack

### DISCUSSION THREAD

**Discussion by Adrianortiz-PR** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#5] [Ideas]
To initiate progress on the project, I suggest we begin by discussing the appropriate programming languages and libraries that will best suit our objectives. This conversation will lay the foundation for our development strategy, ensuring that we select the most effective tools for the task at hand.

**Comment by JeanSanchezFelix** [comment]
I suppose we can just use a basic tech stack of HTML, CSS, and JS without integrating any bloated JS libraries, just so we can make it more accessible for everybody, unless the majority here has previous experience with the same JS library.

**Reply by GabyMarr** [reply]
I wholeheatedly disagree. I think the purpose of this massive project with a big team of developers is that we can leverage other people's experience. I, for one, would love to use this project to learn more different and useful libaries and tools.

**Reply by JeanSanchezFelix** [reply]
That is great, please can list out in a proposal doc a list of libraries for web development that you would like the team to use as a learning experience. Please remember when listing out libraries to tell us the pros and cons of each, the last thing we need is to use a library just because its popular or trending, we need to make data driven decision. As a wrong tech stack would possible set us back weeks if not months. Later a poll can be made to chose a library. :)

**Reply by Ojani** [reply]
I think that before we even talk about tech stack, we need to talk about what will be done in the first place, whether that is a web app or something else.

**Reply by MiguelGibo** [reply]
I agree with Ojani. While I'm sure well need some basic HTML, CSS, and JS, like Jean suggested, we should first think of what the project requirements are and build the stack based on those needs.
